### PR TITLE
Interactive options

### PR DESCRIPTION
**Pull request by jibarnum**
Addresses issues brought up in Issue #64: "Interactive Plot has a few other options in IDL that we don't have". I also fixed pep8 where I saw issues (except for messing with how imports are done, since we need those in a particular order/location for pytplot to work properly). I do apologize for the million and a half pieces of code that got changed... this is why I decided to break the original ticket up before it got worse!

### CODE REVIEW

File: pytplot/tplot.py [hunk 1]
```diff
@@ -194,16 +193,37 @@ def tplot(name,
             exporter.export(save_png)
 
         if display:
+            # Set up all things needed for when a user asks to save plot from window
+            layout_orig = QtPlotter.generate_stack(name, var_label=var_label, auto_color=auto_color,
+                                                   combine_axes=combine_axes,
+                                                   mouse_moved_event=pytplot.hover_time.change_hover_time,
+                                                   crosshair=crosshair)
```
Comment (bryan-harter) [inline pytplot/tplot.py]: Should this crosshair be set to the tplot_options one?  

Does it even matter what this one is set to if the plot is static like this?  Could we get rid of the crosshair parameter tplot()?

**Comment by bryan-harter** [comment]
I think we should turn crosshairs on by default.  

Also, I think that we should remove the dependency between the crosshairs and the interactive plots.  Right now the crosshairs need to be turns on in order for the interactive plot to work, but I think it should work regardless.

**Comment by bryan-harter** [comment]
Other than those comments it looks good!

**Comment by jibarnum** [comment]
@bryan-harter good point - I had forgotten I left the _addmouseevents() to be dependent on crosshairs being turned on. Took that out, should work now!

File: pytplot/QtPlotter/TVarFigureSpec.py [hunk 2]
```diff
@@ -175,45 +171,48 @@ def _mousemoved(self, evt):
...
+            y = np.asarray((specframe.loc[0, 0:speclength - 1]))
+            y_sub = abs(y - index_y * np.ones(y.size))
             y_argmin = np.argmin(y_sub)
             y_closest = y[y_argmin]
             index = int((np.nonzero(y == y_closest))[0])
             dp = dataframe[index][x_closest]
 
             # add crosshairs
             if self._mouseMovedFunction is not None:
+                print('in spec')
```
Comment (bryan-harter) [inline pytplot/QtPlotter/TVarFigureSpec.py]: :eyes:

**Pull request merged**